recipes/mongoose/disable-coverage-clang.patch

```diff
@@ -0,0 +1,30 @@
+From 35305d55ca723b1f13db437e694cc006971c3508 Mon Sep 17 00:00:00 2001
+From: Min RK <benjaminrk@gmail.com>
+Date: Thu, 3 Oct 2019 13:20:53 +0200
+Subject: [PATCH] disable --coverage flag
+
+seems to fail with conda clang
+---
+ CMakeLists.txt | 6 +++---
+ 1 file changed, 3 insertions(+), 3 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 64c09272..322a273c 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -323,9 +323,9 @@ if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang" OR "${CMAKE_CXX_COMPILER_ID}" ST
+     # using Clang
+     SET(CMAKE_CXX_FLAGS "-O3 -fwrapv")
+     # Debug flags for Clang
+-    SET(CMAKE_CXX_FLAGS_DEBUG "--coverage -g -fwrapv")
+-    SET(CMAKE_C_FLAGS_DEBUG "--coverage -g")
+-    SET(CMAKE_EXE_LINKER_FLAGS_DEBUG "--coverage -g")
++    SET(CMAKE_CXX_FLAGS_DEBUG "-g -fwrapv")
```

Comment:
If these are getting used, that means you are building in Debug mode.

Reply:
the build unconditionally does debug-compilation as well for tests, even if building a release